### PR TITLE
Fix for #362

### PR DESCRIPTION
**Pull request by lyneca**
Updated to follow a change in the way Facebook shows user presence on Messenger

### CODE REVIEW

**Comment by Schmavery** [comment]
Thanks for investigating this.  You can delete those lines in the docs rather than commenting them out. 
Are there any other possible values for presence?

**Comment by lyneca** [comment]
Well I would expect that there's a `1` state, possibly meaning invisible or something, but I didn't observe it while I was testing. I don't know how to activate an invisible state anyway so I can't test if that's what it is.

**Comment by Schmavery** [comment]
This seems to be a different fix for the same issue as #330.  Care to comment as to why you think using presence.a is a better option?  Did the other fix not work for you?  Just curious and trying to figure out which one to merge.  Thanks and sorry for the delay in response!

File: DOCS.md [hunk 1]
```diff
@@ -526,7 +526,9 @@ If enabled through [setOptions](#setOptions), this will also return presence, (`
 - `type`: The string "presence".
 - `timestamp`: How old the information is.
 - `userID`: The ID of the user whose status this packet is describing
-- `statuses`: An object with the following fields: `fbAppStatus`, `messengerStatus`, `otherStatus`, `status` and `webStatus`. All can contain any of the following values: `"active"`, `"idle"`, `"invisible"`, `"offline"`.
+<!-- The form of the status request response has changed; it no longer sends five seperate keys - instead it sends a single value. The status states have been condensed to three numbers. -->
+<!-- - `statuses`: An object with the following fields: `fbAppStatus`, `messengerStatus`, `otherStatus`, `status` and `webStatus`. All can contain any of the following values: `"active"`, `"idle"`, `"invisible"`, `"offline"`. -->
```
Comment (Schmavery) [inline DOCS.md]: If you could delete this I'd be super happy to merge :)

Comment (lyneca) [inline DOCS.md]: Delete the commented-out section?

**Comment by lyneca** [comment]
I didn't even see #330 :P 

`presence.a` is just what facebook returns now - I'm not really sure what the fix in #330 actually does.

File: DOCS.md (hunk 1 above)
Comment (Schmavery) [inline DOCS.md]: Yeah, no point in keeping it around if we're changing it :)

**Comment by Schmavery** [comment]
Haha ok, sounds good.  Facebook changes so fast it can be hard to keep up.

**Comment by lyneca** [comment]
I guess that's the problem with writing an unofficial API for a service that's not built for public access 😛

**Comment by Schmavery** [comment]
Thanks!

**Pull request merged**